e2e-tests.sh

```diff
@@ -178,7 +178,7 @@ function initialize() {
   done
 
   if [[ "${CLOUD_PROVIDER}" == "gke" ]]; then
-      custom_flags+=("--addons=NodeLocalDNS")
+      custom_flags+=("--command-group=") # Use production GKE commands, instead of beta.
```

Comment:
When https://github.com/knative/test-infra/pull/3623 is merged this might be removed.

Reply:
Removed in 5af6017e0b2bdb5653977c71f508e3e82940ad36